www/_template/reference/configuration.md

```diff
@@ -124,211 +337,239 @@ module.exports = {
+Emit source maps for installed packages.
 
-- When using the Single-Page Application (SPA) pattern, this is the HTML "shell" file that gets served for every (non-resource) user route. Make sure that you configure your production servers to serve this as well.
+**_Experimental:_** Set to `true` to enable source maps.
 
-#### devOptions.open | `string` | Default: `"default"`
+### installOptions.env
 
-- Opens the dev server in a new browser tab. If Chrome is available on macOS, an attempt will be made to reuse an existing browser tab. Any installed browser may also be specified. E.g., "chrome", "firefox", "brave". Set "none" to disable.
+**Type**: `{[ENV_NAME: string]: (string true)}`
 
-#### devOptions.output | `"stream" | "dashboard"` | Default: `"dashboard"`
+Sets a `process.env.` environment variable inside the installed dependencies.
 
-- Set the output mode of the `dev` console.
-- `"dashboard"` delivers an organized layout of console output and the logs of any connected tools. This is recommended for most users and results in the best logging experience.
-- `"stream"` is useful when Snowpack is run in parallel with other commands, where clearing the shell would clear important output of other commands running in the same shell.
+If set to true (ex: `{NODE_ENV: true}` or `--env NODE_ENV`) this will inherit from your current shell environment variable. Otherwise, set to a string (ex: `{NODE_ENV: 'production'}` or `--env NODE_ENV=production`) to set the exact value manually.
+
+### installOptions.rollup
+
+**Type**: `Object`
+
+Allows customization of Snowpack's internal Rollup configuration.
+
+Snowpack uses Rollup internally to install your packages. This `rollup` config option gives you deeper control over the internal Rollup configuration that we use.
+
+- installOptions.rollup.plugins | `TODO I'm not sure the best way to specify this, it appears to be an array of objects? an example would be useful` - Specify Custom Rollup plugins if you are dealing with non-standard files.
+- installOptions.rollup.dedupe | `array[string]` - If needed, deduplicate multiple versions/copies of a packages to a single one. This helps prevent issues with some packages when multiple versions are installed from your node_modules tree. See [rollup-plugin-node-resolve](https://github.com/rollup/plugins/tree/main/packages/node-resolve#usage) for more documentation.
+- installOptions.rollup.context | `string` - Specify top-level `this` value. Useful to silence install errors caused by legacy common.js packages that reference a top-level this variable, which does not exist in a pure ESM environment. Note that the `'THIS_IS_UNDEFINED'` warning ("'this' keyword is equivalent to 'undefined' ... and has been rewritten") is silenced by **Default**, unless `--verbose` is used.
+
+### installOptions.namedExports
+
+**Type**: `string[]`
+
+Import CJS packages using named exports
+
+TODO I don’t see in types.ts or config.ts
```

Comment:
Just curious why it isn't in these files? @FredKSchott

Reply:
install options come directly from `esinstall` so we try to import them from there instead of redefining them ourselves. You can see this in the `import type {InstallOptions} from 'esinstall';` line of types.ts.